app.js

```diff
@@ -943,44 +943,41 @@ define(function(require) {
 		prepareReviewData: function(data) {
 			var self = this,
 				expected = _.get(data, 'columns.expected', []),
+				modifierPerProp = {
+					brand: _.toLower,
+					family: _.toLower,
+					model: _.toLower
+				},
 				formattedData = {
 					data: {
 						fileName: data.fileName,
 						totalRecords: data.records.length,
 						columns: {
 							actual: data.columns.actual,
-							expected: expected
+							expected: expected,
+							others: []
 						},
-						recordsToReview: data.records.slice(0, 5),
+						recordsToReview: data.isDevices ? _.map(data.records, function(record) {
+							return _.chain(record)
+								.merge(record, _.reduce(record, function(row, prop, index) {
+									if (modifierPerProp[index]) {
+										row[index] = modifierPerProp[index](prop);
+									}
+									return record;
+								}))
```

Comment:
Here we can abstract the if statement by using `lodash#identity`, which returns the first argument it receives:
```suggestion
									var modifier = _.get(modifiersPerProp, prop, _.identity);

									return _.set(acc, prop, modifier(value));
								}, {}))
```

Don't forget to give reduce an initial value (accumulator).